sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/tests/Scenario/VirtualApplicationCollectionTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.DesktopVirtualization.Models;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.DesktopVirtualization.Tests.Tests
+{
+    public class VirtualApplicationCollectionTests : DesktopVirtualizationManagementClientBase
+    {
+        public VirtualApplicationCollectionTests() : base(true)
+        {
+        }
+
+        public VirtualApplicationCollectionTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        public VirtualApplicationCollectionTests(bool isAsync, RecordedTestMode mode) : base(isAsync, mode)
+        {
+        }
+
+        [Test]
```

Comment:
Can we apply this change to all the tests? We want to do this because the `RecordedTest` attribute will help us automatically re-record the failed tests in playback mode.
```suggestion
        [TestCase]
        [RecordedTest]
```